skimage/restoration/tests/test_denoise.py

```diff
@@ -144,6 +144,12 @@ def test_denoise_bilateral_3d():
     assert out1[30:45, 5:15].std() > out2[30:45, 5:15].std()
 
 
+def test_denoise_bilateral_nan():
+    img = np.NaN + np.empty((50, 50))
```

Comment:
Here you read uninitialized data. Perhaps you meant np.zeros instead of np.empty?